docs/guides/primitive-input-output.ipynb

```diff
@@ -0,0 +1,284 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e8a8a4b1-2ed2-4b82-8d02-98174a6da903",
+   "metadata": {},
+   "source": [
+    "# Understanding primitive inputs and outputs\n",
+    "\n",
+    "This page gives an overview of the inputs and outputs of the Qiskit Runtime V2 primitives that execute workloads on IBM Quantum&trade; compute resources. These primitives provide you with the ability to efficiently define vectorized workloads by using a data structure known as a **Primitive Unified Bloc (PUB)**. These PUBs are the fundamental unit of work a QPU needs to execute these workloads.\n",
+    "\n",
+    "After being submitted as jobs to the IBM Quantum Platform, they are executed and the data returned is dependent on the format PUBs which were executed, as well as any error mitigation, suppression, or other runtime options that were defined when the primitive was invoked."
```

Comment:
"dependent on the format PUBs which were executed..." sounds a bit off to me? is there a typo?

Reply:
We also need to remove the "the" before IBM Quantum Platform. Would this work better?:

     "After being submitted as jobs to IBM Quantum Platform, they are executed and the data returned is dependent on the format of the PUBs that were executed"

Reply:
Hopefully I addressed this! Feel free to give it another read.